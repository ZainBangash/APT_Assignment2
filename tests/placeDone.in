2
TEST2
place Done
