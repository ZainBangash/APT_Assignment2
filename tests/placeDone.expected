Welcome To Scrabble!
------------------- 
Menu
----
1. New Game
2. Load Game        
3. Credits
4. Quit
1

Starting a New Game

Enter a name for player 1 (uppercase characters only)
TANIA

Enter a name for player 2 (uppercase characters only)
ZAIN


Player 1: TANIA, 0
Player 2: ZAIN, 0

Lets Play!

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
I-1 R-1 E-1 J-8 E-1 I-1 R-1
Enter command: save TEST2


Game saved successfully

Welcome To Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit
2

Enter the filename from which load a game
TEST2
     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
I-1 R-1 E-1 J-8 E-1 I-1 R-1 I-1 R-1 E-1 J-8 E-1 I-1 R-1
Enter command: place Done

A-1 O-1 E-1 D-2 D-2 C-3 A-1 A-1 O-1 E-1 D-2 D-2 C-3 A-1
 TANIA has 0 points
 ZAIN HAS 0 points